### PR TITLE
Clean up deprecated usage of router-id

### PR DESCRIPTION
**Pull request by donaldsharp**
Signed-off-by: Donald Sharp <sharpd@cumulusnetworks.com>

### CODE REVIEW

**Comment by donaldsharp** [comment]
@mwinter-osr @rzalamena how are we supposed to do this right?

**Comment by louberger** [comment]
In the current setup, different configs require different tests. You can 
see the new rfapi..._config2 for an example....


----------

File: bgp_rfapi_basic_sanity/test_bgp_rfapi_basic_sanity.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 def test_add_routes():
     CliOnFail = None
     # For debugging, uncomment the next line
-    #CliOnFail = 'tgen.mininet_cli'
+    CliOnFail = 'tgen.mininet_cli'
```
Comment (louberger) [inline bgp_rfapi_basic_sanity/test_bgp_rfapi_basic_sanity.py]: clionfailure shouldn't be checked in

Comment (louberger) [reply bgp_rfapi_basic_sanity/test_bgp_rfapi_basic_sanity.py]: this file should be removed from the PR

**Comment by rzalamena** [comment]
This was fixed in https://github.com/FRRouting/topotests/pull/132 .

**Pull request closed without merging**